[New Term entry] added getDisplayNames.md (#2995)
Codecademy/docs: content/java/concepts/calendar/terms/getDisplayNames/getDisplayNames.md

---
Title: '.getDisplayNames()'
Description: 'Returns a map containing all string representations of Calendar field values in the given style and locale.'
Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Date'
  - 'Methods'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> The `.getDisplayNames()` method of the `Calendar` class returns a map containing all string representations of Calendar field values in the given style and locale and the corresponding field values.

Reviewer: ```suggestion
The `.getDisplayNames()` method of the `Calendar` class returns a map containing the string representations of the field values in the given style and locale.
```

## Syntax

```pseudo
calendar.getDisplayNames(field, style, locale)
```

Where:

- `field`: The calendar field for which the display names are returned. For example, `Calendar.MONTH`.
- `style`: The style applied to the display names. For example, `Calendar.LONG`.
- `locale`: The locale for which the display names are returned. For example, `Locale.US`.

It can return `null` if there are no display names for the given field and style in the given locale.

> Throws [NullPointerException](https://www.codecademy.com/resources/docs/java/errors/nullpointerexception) if `locale` is `null`.

Reviewer: ```suggestion
The function will throw a [`NullPointerException`](https://www.codecademy.com/resources/docs/java/errors/nullpointerexception) if `locale` is `null`.
```

## Example

> The following example, `.getDisplayNames()` method returns a map containing all days of the week in short form for the US locale. The days are paired with integer values representing the index of the day in the week (Sunday is 1, Monday is 2, etc.)

Reviewer: ```suggestion
In the following example, `.getDisplayNames()` method returns a map containing all the days of the week in short form for the US locale. The days are paired with integer values representing the index of the day in the week (Sunday is 1, Monday is 2, etc.)
```
